### PR TITLE
Add all public servers

### PR DESCRIPTION
**Pull request by paulzierep**
Since the querying of the public servers takes a while, when they time out and such, I put the query in a separate CI, 
this makes testing much easier. This creates a TSV that is then used by extract_galaxy_tools.py to add availability on all servers.

### CODE REVIEW

**Comment by paulzierep** [comment]
Still needs to be merged with: https://github.com/galaxyproject/galaxy_tool_metadata_extractor/pull/112

File: bin/extract_galaxy_tools.py [hunk 1]
```diff
@@ -28,15 +28,16 @@
 # BIOTOOLS_API_URL = "https://130.226.25.21"
 
 USEGALAXY_SERVER_URLS = {
-    "UseGalaxy.org": "https://usegalaxy.org",
+    "UseGalaxy.org (Main)": "https://usegalaxy.org",
```
Comment (bebatut) [inline bin/extract_galaxy_tools.py:31]: I would remove `Main` here

Comment (paulzierep) [reply bin/extract_galaxy_tools.py:31]: it is the name you get from https://galaxyproject.org/use/feed.json , so I use this one to avoid dublication

**Comment by paulzierep** [comment]
can we merge it @bebatut ?

**Pull request merged**